src/tools/bin/plot_chargeConservation_overTime.py

```diff
@@ -83,12 +83,7 @@ def deviation_charge_conservation(series, iteration):
             norm = np.max([norm, np.amax(np.abs(species_Density))])
             # add charge density to total charge density
             charge += species_Density
-            # We check the attribute _size of any/all density_[species].
-            # libSplash always keeps this as an array of length 3. It
```

Comment:
This comment should stay except that: `libSplash` should be replaced by `openPMD-api HDF5`